lib/smart_proxy_remote_execution_ssh/runners/script_runner.rb

```diff
@@ -102,6 +102,8 @@ class ScriptRunner < Proxy::Dynflow::Runner::Base
     EXPECTED_POWER_ACTION_MESSAGES = ['restart host', 'shutdown host'].freeze
     DEFAULT_REFRESH_INTERVAL = 1
     MAX_PROCESS_RETRIES = 4
+    VERIFY_HOST_KEY = Gem.loaded_specs['net-ssh'].version < Gem::Version.create('5.0.0') ||
```

Comment:
Can `Gem.loaded_specs` be empty? I'm not sure about production RPM deployments.

Reply:
Not sure, changed.